src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -6,43 +6,47 @@
 import com.google.cloud.bigquery.TableResult;
 import java.io.IOException;
 
+
 public class GcpClient {
 
-    public static int countBqQuery(String table) throws IOException, InterruptedException {
+    public  int countBqQuery(String table) throws IOException, InterruptedException {
 
         String projectId = SeleniumHelper.readParameters("Project-ID");
         String datasetName = SeleniumHelper.readParameters("Data-Set");
-        String query = "SELECT count(*) "
+        String selectQuery = "SELECT count(*) "
                 + " FROM `"
                 + projectId
                 + "."
                 + datasetName
                 + "."
                 + table
                 + "`";
-        BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
-        QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(query).build();
-        TableResult results = bigquery.query(queryConfig);
-        return Integer.parseInt((String) results.getValues().iterator().next().get(0).getValue());
+        return executeQuery(selectQuery);
     }
     //Deleting the table
-    public static void dropBqQuery(String table) throws IOException, InterruptedException {
+    public  void dropBqQuery(String table) throws IOException, InterruptedException {
 
         String projectId = SeleniumHelper.readParameters("Project-ID");
         String datasetName = SeleniumHelper.readParameters("Data-Set");
-        String query = "DROP TABLE `"
+        String dropQuery = "DROP TABLE `"
                 + projectId
                 + "."
                 + datasetName
                 + "."
                 + table
                 + "`";
+        executeQuery(dropQuery);
+    }
 
-        BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+    public int executeQuery(String query) throws InterruptedException, IOException {
+        BigQuery bigquery =  BigQueryOptions.newBuilder().setProjectId(SeleniumHelper.readParameters("Project-ID"))
+                .build().getService();
         QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(query).build();
-        TableResult results = bigquery.query(queryConfig);
-
-
+       TableResult results = bigquery.query(queryConfig);
+       if (results.getTotalRows()>0) {
+         return Integer.parseInt((String) results.getValues().iterator().next().get(0).getValue());
```

Comment:
what does this method return? Can you add a comment to clarify?

Reply:
Also I believe instead of doing `getValue` and parsing you can just call `getLongValue`

Reply:
Added comment and no of records present should be int so parsed it in integer